### ISSUE TITLE
exporting to OpenAPI v3 makes empty content

### ISSUE THREAD

**Issue by sayjeyhi** [mockoon/mockoon#352]
**Describe the bug**
I have an environment with about 50 routes and every route has a success and failure response.
When I try to export the environment to an OpenAPI v3 version, the generated file does not contain the 
routes responses and just have `"content":{"application/json":{}}` like below:

```json
{
"paths": {
    "/auth/login": {
      "post": {
        "description": "Login user",
        "responses": {
          "200": {
            "description": "Success login",
            "content": {
              "application/json": {}
            },
            "headers": {}
          },
          "400": {
            "description": "Failed login",
            "content": {
              "application/json": {}
            },
            "headers": {}
            }
          }
        }
      }
    },
}
```

**To Reproduce**
It exists even in the sample testing environment.

**Expected behavior**
I think you may don't add the response to OpenAPI content because of templating feature.
But if someone(like me) did not use templating, or just used faker and wants to make a `swagger` or `postman` collection with Mockoon, this will be a real need.

**Screenshots**

**Mockoon version:**
1.10.0

**OS / OS version:**
Mac OS

**Comment by sayjeyhi** [comment]
for now, I applied a few changes to `convertToOpenAPIV3` method and fixed my export.
the detail could be accessible here:
https://github.com/sayjeyhi/mockoon/commit/d489ff19a953cf0af27bca4390ff5e9daa19403c

**Comment by 255kb** [comment]
Hi @sayjeyhi thank you for opening this issue. It's in fact not a bug but a missing feature. 
It seems that we can easily use the `examples` property to add this. I expected having to deal with schemas and definitions but it may be easier.

**Comment by mauriceackel** [comment]
I checked a bit on this and the relevant documentation can be found [here](https://swagger.io/docs/specification/2-0/adding-examples/?sbsearch=Examples), namely under the heading "Whole Schema Examples".

I tried it a bit in the editor and it seems that the following is valid OpenAPI (v3) syntax:

```
application/json:
  schema:
    example:
      foo: "bar"
      array:
      - "a"
      - "b"
```

which suggests that there would be no need to also export the real schema (i.e. `foo: string`, etc.).

However, one issue I see here is the use of handlebar syntax, as this will most probably not result in valid JSON syntax at "design time" (i.e. when it is not interpreted by the handlebar runtime, as it would be when exporting it). Unfortunately I see no good way to circumvent this besides not exporting an example if the body is not valid JSON.

In my personal opinion, I would not include this feature into Mockoon, as I think it would be messy to have a feature that is not supposed to work with the tempting syntax. Furthermore, I think Mockoon is an API mocking tool in the first place and there are probably better suited tools for the purpose of complete and proper API documentation. As a result, I would also not pursue the feature request in #354 .

**Comment by 255kb** [comment]
You are right @mauriceackel that Mockoon is primarily a prototyping/mocking tool and should not be considered as an API design tool like Stoplight. 
Still, I think that we could create a schema from a body when exporting. Only thing would be to detect when templating is used. For this we don't have a reliable way. We can only check for the presence of helpers `{{...}}` and if the JSON is valid.

**Comment by martsim6** [comment]
Hello, I have same issue when trying to **Export current environment to OpenAPI v3 (JSON)**. Output contains only routes with empty object as response, even all routes has defined response body.

BUT - when I use Right click on actual env. & choose option **Copy configuration to clipboard (JSON)** the clipboard contains correct configuration with all routes & responses.

For now, I can use this as 'export', but I'm not sure if it's wanted behavior. 
Using Ubuntu 22, mockoon installed via snap.

Thanks!

**Comment by 255kb** [comment]
@martsim6 What you got when right clicking is the Mockoon data file, which is very different from an OpenAPI spec. You can read an explanation [here](https://mockoon.com/docs/latest/openapi/openapi-specification-compatibility/).

TL;DR: Mockoon stores everything in JSON files (what you get when doing a right click). You are supposed to use this with the CLI because it saves every single settings from the UI. When you export/import from/to OpenAPI, you lose information, it's only a starting point. (Mockoon is not an OpenAPI editor)

**Comment by martsim6** [comment]
@255kb Thanks for your response. I understand your point, but in my case I don't need OpenAPI spec, I just need to export actual env (in any format) and import it back on another HW (or share with others so they have same env. in their mockoon). 

That's why I can use right-click clipboard data, because I can share this content to other HW, so the env is always up to date on all devices using this env.

Ofc, it would be way easier, if I could only export it, upload file and then just import uploaded file from diff HW, because now I have to modify input file and replace content with clipped one to keep it up to date.

**Comment by 255kb** [comment]
@martsim6 
I think you misunderstood how Mockoon is working. 
The app can open multiple "environments" (or data files) at once. What you describe in "1." is working when clicking the "Open environment" button:
![image](https://user-images.githubusercontent.com/7489814/229791595-f11a8827-a1d1-43f0-a03e-93ac350159f0.png)

You don't need to move anything in the default storage folder, a data file can be opened from anywhere. 

Also, your issue is not related to the one here, which is about exporting in OpenAPI format. If you have more questions I would be glad to help. I would just kindly ask you to use [GitHub discussions](https://github.com/mockoon/mockoon/discussions).

**Comment by martsim6** [comment]
@255kb agree, that's exactly what I was looking for. Just I would expect this exact behavior from Import... MB than. 

Thanks!

**Comment by 255kb** [comment]
📦 This is now available with [release v4.0.0](https://mockoon.com/releases/4.0.0).
